packages/google_maps_flutter/google_maps_flutter_android/android/src/main/java/io/flutter/plugins/googlemaps/Convert.java

```diff
@@ -223,11 +199,11 @@ public static BitmapDescriptor getBitmapFromBytes(
    * 'imagePixelRatio', and may optionally include 'width' and/or 'height' to explicitly set the
    * dimensions of the output image.
    *
-   * @param assetDetails a map containing the asset details and scaling instructions, with keys
-   *     'assetName': the name of the asset file, 'bitmapScaling': the scaling mode, either 'auto'
-   *     or 'none', 'imagePixelRatio': used with 'auto' scaling to compute the scale ratio, 'width'
-   *     (optional): the desired width, which affects scaling if 'height' is not provided, 'height'
-   *     (optional): the desired height, which affects scaling if 'width' is not provided
+   * @param assetMap a [PlatformBitmapAssetMap] containing the asset name from which to construct a
+   *     [BitmapDescriptor] and a bitmap scaling mode. If the scaling mode is `AUTO`, `bytesMap`
```

Comment:
You don't need to list everything in the class; that was only necessary because it was a map, so it could have had anything. The type itself expresses most of this now.

The important part is about how height/width works.